dev/integration_tests/ios_add2app_life_cycle/flutterapp/pubspec.yaml

```diff
@@ -31,6 +31,8 @@ dependencies:
   vector_math: 2.1.4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
 
 dev_dependencies:
+  integration_test:
+    sdk: flutter
```

Comment:
This adds a dev dependency with platform-specific code. This is needed for the integration tests that verify that the dev dependency check works on Flutter iOS modules.